feat(dashboards): revise custom dashboards to include manage (#3558)
getsentry/sentry-docs: src/docs/product/dashboards/custom-dashboards/index.mdx

</Note>

Custom Dashboards are shared across your organization, and users can view each other’s custom dashboards in the management view. Note that any changes you make are global, and will appear for users across the entire organization.

## Manage Dashboards

> When you have either a Business or Trial Plan, you'll land on a view to manage dashboards instead of the [default dashboard](/product/dashboards/). From here, you can:

Reviewer: ```suggestion
When you have either a Business or Trial Plan, navigating to **Dashboards** takes you to a view where you can manage dashboards instead of the [default dashboard](/product/dashboards/). From here, you can:
```

- Create a dashboard
- Duplicate a dashboard
- Delete a dashboard

![Manage mode.](dashboard-management.png)

> Clicking "Create Dashboard" will immediately take you to a new dashboard where you can immediately begin adding widgets. When you're creating a new dashboard or edit an existing dashboard, the widgets will be slightly grayed out. From here, you can:

Reviewer: I've added a break between the sentence about creating a dashboard and the sentence about creating or editing because the list that comes after it pertains to both create and edit. It's confusing to have it preceded by a paragraph that starts with just creating.

```suggestion
Clicking "Create Dashboard" takes you to a new dashboard where you can immediately begin adding widgets. 

When you're creating a new dashboard or editing an existing one, the widgets are slightly grayed out. From here, you can:
```

- Rename the dashboard
- Add and remove widgets
- Rearrange existing widgets

![Edit mode.](dashboard-edit.png)

## Widget Builder

When you add a widget or edit an existing one, you’ll be prompted with a form and preview of the visualization. From here, you can:
- Rename the widget
- Change the queries
- Provide legend aliases
- Change the Y-Axes

> ![Widget builder.](dashboard-widget-builder.png)

Reviewer: Should add a sentence after the widget builder image that precedes the list. Something like:

The widget form includes these fields:
